### ISSUE TITLE
ENOENT No bower.json present

### ISSUE THREAD

**Issue by chintansoni202** [Polymer/polymer-cli#197]
I am trying learn with my first polymer app and I am stuck.

I started from here: https://www.polymer-project.org/1.0/start/toolbox/set-up

I installed the Polymer CLI successfully. Now, When I am trying to initialize project from a template, it shows following log in my cmd:

```
C:\Users\chint\Desktop\polymer-app>polymer init app-drawer-template
info:    Running template app-drawer-template
info:    Downloading latest release of Polymer/app-drawer-template
info:    Unpacking template files
info:    Finished writing template files

I'm all done. Running bower install for you to install the required dependencies. If this fails, try running the command yourself.


bower                           ENOENT No bower.json present
```

And when I checked, my polymer-app folder was empty. Nothing got dowloaded.

Can anyone please help me with it ?

PS: my node.js version is 6.1.0

**Comment by naoak** [comment]
Similar to #167?

**Comment by thibaultsavary** [comment]
Hey,
Same error here.

Windows 10, Node 6.2.0 & Polymer-cli 0.8.0

Similar to [#167#issuecomment-220668841](https://github.com/Polymer/polymer-cli/issues/167#issuecomment-220668841)

**Comment by tennosys** [comment]
Try to fix it with my changes here
[57a77a40d8e6d80e071d37d1e1f31bec74d47918](https://github.com/Polymer/polymer-cli/pull/188/commits/57a77a40d8e6d80e071d37d1e1f31bec74d47918)

**Comment by thibaultsavary** [comment]
@tennosys Thank you, it did the trick.

**Comment by FredKSchott** [comment]
This has been fixed in #199 (I can no longer reproduce). Please install the newest version on npm (0.9.0) to get the fix.
